frontend/packages/ceph-storage-plugin/src/components/ocs-install/ocs-request-data.ts

```diff
@@ -71,6 +76,21 @@ export const labelNodes = (selectedNodes: NodeKind[]): Promise<NodeKind>[] => {
   );
 };
 
+export const labelNamespace = (name: string): Promise<K8sKind> =>
+  k8sPatch(
+    NamespaceModel,
+    {
+      metadata: {
+        name,
```

Comment:
I shall avoid passing dynamic namespace when we are labeling just a single namespace for ocs - `CEPH_STORAGE_NAMESPACE`